landcover.mss

```diff
@@ -546,13 +527,18 @@
 }
 
 #landuse-overlay {
-  [landuse = 'military'][zoom >= 10]::landuse {
+  [feature = 'military'][zoom >= 10] {
     polygon-pattern-file: url('symbols/military_red_hatch.png');
     polygon-pattern-alignment: global;
     line-color: @military;
     line-width: 3;
     line-opacity: 0.329;
   }
+  [feature = 'forest'][zoom >= 12]  {
+    polygon-pattern-file: url('symbols/forest.png'); // Lch(50,30,135)
+    polygon-pattern-alignment: global;
+    opacity: 0.6; // The entire layer has opacity in case of overlapping forests
```

Comment:
They overlap at 100% opacity and then are reduced to 60%. This might result in some anti-aliasing artifacts if you look *very* closely - the edges of the symbols aren't 100% opacity in the PNG - but I couldn't see any issues.

If polygon-pattern-opacity were used then overlapping forests would have stronger symbols.